backend/AsterixUtils.py

```diff
@@ -24,7 +24,7 @@ def executePeterListQuery(query):
 def queryAsterix(query):
 	response = executePeterListQuery(query)
 	if response != None:
-		return json.dumps(json.loads(response.text)['results'])
+		return json.dumps(json.loads(response.text))#['results'])
```

Comment:
Is this change intentional?

Reply:
Yes. I tested other parts of the code that used it function.